schedule/rescue_system_sle11sp4_staging.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name:           rescue_system_sle11sp4
```

Comment:
It is only running in staging, but let's rename the file to `rescue_system_sle11sp4.yaml`

Reply:
also done